dev/sg/sg_setup_ubuntu.go

```diff
@@ -199,6 +199,24 @@ asdf plugin-add rust https://github.com/asdf-community/asdf-rust.git
 asdf install rust
 `,
 			},
+			{
+				name:                 "protoc",
+				check:                check.InPath("protoc"),
+				instructionsComment:  `Souregraph requires protoc to be installed.`,
+				instructionsCommands: `apt install -y protobuf-compiler`,
+			},
+			{
+				name:                 "protoc-gen-go",
+				check:                check.InPath("protoc-gen-go"),
+				instructionsComment:  `Souregraph requires protoc-gen-go to be installed.`,
+				instructionsCommands: `go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28`,
```

Comment:
That's going to break on install with `asdf`, because the shims won't be there after the install, here is a hacky fix:

```suggestion
				instructionsCommands: `go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28 && if hash asdf 2>/dev/null; then asdf reshim golang; fi`,
```